server/auth/types/saml/saml_auth.ts

```diff
@@ -130,6 +130,11 @@ export class SamlAuthentication extends AuthenticationType {
     return {};
   }
 
+  // JWT auth types should get expiry from the JWT, and not override this value
```

Comment:
you mean SAML here?

Reply:
Here I am considering SAML to fundamentally be similar to JWT auth types, but agree this can be confusing. Resolving the comment since the implementation changed.